src/com/synopsys/integration/pipeline/versioning/GithubReleaseStage.groovy

```diff
@@ -1,172 +1,92 @@
         super(pipelineConfiguration, stageName)
-        this.runRelease = runRelease
-        this.artifactFile = null
-        this.artifactPattern = artifactPattern
-        this.artifactDirectory = artifactDirectory
-        this.branch = branch
+        this.releaseOwner = releaseOwner
+        this.releaseRepo = releaseRepo
+        this.githubCredentialsId = githubCredentialsId
     }
 
     @Override
     void stageExecution() throws PipelineException, Exception {
-        if (!runRelease) {
-            getPipelineConfiguration().getLogger().info("Skipping the ${this.getClass().getSimpleName()} because this is not a release.")
-            return
-        }
-        String version = getPipelineConfiguration().getScriptWrapper().getJenkinsProperty(GITHUB_RELEASE_VERSION)
-        if (StringUtils.isBlank(version)) {
-            throw new PrepareForReleaseException("Could not find the \"${GITHUB_RELEASE_VERSION}\" environment variable. Will not perform the GitHub release.")
-        }
-        List<String> options = []
-        options.add('-o')
-        options.add(owner)
-        options.add('-v')
-        options.add(version)
-        if (StringUtils.isNotBlank(artifactFile)) {
-            options.add('-f')
-            options.add(artifactFile)
-        }
-        if (StringUtils.isNotBlank(artifactPattern)) {
-            options.add('-t')
-            options.add(artifactPattern)
-        }
-        if (StringUtils.isNotBlank(artifactDirectory)) {
-            options.add('-d')
-            options.add(artifactDirectory)
-        }
-        if (StringUtils.isNotBlank(project)) {
-            options.add('-p')
-            options.add(project)
-        }
-        options.add('-m')
-        options.add("\"${releaseDescription}\"")
-
-        GithubBranchParser githubBranchParser = new GithubBranchParser()
-        GithubBranchModel githubBranchModel = githubBranchParser.parseBranch(branch)
-
-        options.add('-br')
-        options.add(githubBranchModel.getBranchName())
+        try {
+            String timeStamp = new SimpleDateFormat("yyyy.MM.dd.HH.mm.ss").format(new Date())
+            setReleaseTagName(getPipelineConfiguration().getScriptWrapper().getJenkinsProperty(GithubReleaseStageLegacy.GITHUB_RELEASE_VERSION))
+            setReleaseBody("Released from Jenkins " + timeStamp)
 
-        String commandOptions = options.join(' ')
+            targetCommitish = getPipelineConfiguration().getScriptWrapper().getJenkinsProperty(RemoveSnapshotStage.RELEASE_COMMIT_HASH)
+            String stringCommandLines = "curl -s -X POST -H \"Accept: application/vnd.github.v3+json\" https://api.github.com/repos/${getReleaseOwner()}/${getReleaseRepo()}/releases -d '{\"tag_name\":\"${getReleaseTagName()}\", \"target_commitish\":\"${getTargetCommitish()}\", \"name\":\"${getReleaseTagName()}\", \"body\":\"${getReleaseBody()}\", \"draft\":false, \"prerelease\":false, \"generate_release_notes\":false}'"
```

Comment:
Are any of the options we are passing (draft/prerelease/generate_release_notes) already set false by default? If yes, can we remove them?

Reply:
Yes, they're default false so I removed those three from the command.